functional_go_test.go

```diff
@@ -0,0 +1,43 @@
+package functional_go
+
+import "testing"
+
+func TestMapDoublingIntegers(t *testing.T) {
+	function_which_doubles_an_integer := func(number int) int {
+		return number * 2
+	}
+
+	integers_to_double := []int{1, 2, 3}
+
+	doubled_integers := Map(
+		integers_to_double,
+		function_which_doubles_an_integer,
+	)
+
+	if len(doubled_integers) != 3 {
+		t.Fatalf(`len(%q) != 3`, doubled_integers)
```

Comment:
I notice when this fails, it doesn't print the slice correctly (or - as I'd like it to be shown); each item is shown something like:

```go
functional_go_test.go:18: len(['\x02' '\x04']) != 3
```

How can I print that more clearly?